common/djangoapps/third_party_auth/tests/test_views.py

```diff
@@ -53,8 +55,8 @@ def test_custom_contact_info(self):
         self.enable_saml(
             other_config_str=(
                 '{'
-                '"TECHNICAL_CONTACT": {"givenName": "Jane Tech", "emailAddress": "jane@example.com"},'  # pylint: disable=unicode-format-string,line-too-long
```

Comment:
I ran PyLint locally, and it flagged these as unnecessary, so I cleaned them up.